airbyte-ci/connectors/pipelines/pipelines/consts.py

```diff
@@ -41,7 +41,7 @@
 DOCKER_CLI_IMAGE = f"docker:{DOCKER_VERSION}-cli"
 DOCKER_REGISTRY_MIRROR_URL = os.getenv("DOCKER_REGISTRY_MIRROR_URL")
 DOCKER_REGISTRY_ADDRESS = "docker.io"
-DOCKER_VAR_LIB_VOLUME_NAME = "docker-cache"
+DOCKER_VAR_LIB_VOLUME_NAME = "docker-cache-2"
```

Comment:
Are we in the position where we should make a directory per day, so that this self heals?